stories/StudyCardView01.svelte

```diff
@@ -0,0 +1,81 @@
+<script>
+  import StudyCard from "../src/StudyCard.svelte";
+
+  const studies = [
+    {
+      case: "Not enrolled in either ion nor study",
```

Comment:
Hm pairing "either" and "nor" breaks my brain :) This maybe:
```suggestion
      case: "Enrolled in neither Ion nor study",
```

Reply:
"Enrolled in neither" isn't super fun to look at either... I think "Not enrolled in Ion or study" would be simpler and clearer still.